pkg/codegen/go/sdk.go

```diff
@@ -10,7 +10,7 @@ func (GoSDK) FolderName() string {
 
 func (GoSDK) Files(pipelineSchema schema.PipelineSchema) map[string]string {
 	return map[string]string{
-		"enviornment.go":  newEnvironmentFile(pipelineSchema.Environment),
+		"environment.go":  newEnvironmentFile(pipelineSchema.Environment),
```

Comment:
@wooly The file name needed to be updated in the codegen package since it gets blown away and recreated each time. I was hopeful my naive check for changes would have caught this... Sorry that wasn't clear!

Reply:
Ah yes, of course. Will fix this up shortly :+1: